scripts/create-releases.sh

```diff
@@ -60,12 +60,12 @@ static_binary_name=soci-snapshotter-${release_version}-linux-${ARCH}-static.tar.
 
 make build
 cp "$NOTICE_FILE" "$LICENSE_FILE" "${OUT_DIR}"
-tar -czvf "$RELEASE_DIR"/"$dynamic_binary_name" "$OUT_DIR"
+tar -czvf "$RELEASE_DIR"/"$dynamic_binary_name" -C "$OUT_DIR" .
```

Comment:
I think it would be better to just tar the files instead of the directory (that's what we did in previous releases).

To fix #1010 as well, we should probably `pushd` into the release dir, create the archive with `tar *`, create the sha256, move both to the release dir, `popd`.

Reply:
can we just `cd $RELEASE_DIR && shas256sum ...` ? I think that would work too.

Reply:
It does work. I don’t think there’s any reason we would have hidden files in release that would be picked up by archiving the whole folder so it should be equivalent.

Can we at least use `pushd` and `popd` so the script doesn’t leave us in the release folder?

Reply:
Sure. Lemme update the pr that way